fulp_modules/main_features/bloodsuckers/code/mobs/clawmonster.dm

```diff
@@ -0,0 +1,14 @@
+/mob/living/simple_animal/hostile/retaliate/tzimisce/clawmonster
+	name = "clawed monster"
+	icon_state = "tarantula"
+	icon_living = "tarantula"
+	icon_dead = "tarantula_dead"
+	maxHealth = 100
```

Comment:
btw, have you considered how members of this new clan are intended to heal their minions? I'd imagine that they'd burn through sutures pretty fast if they tried to use those, and nanites aren't developed every round.